ethclient/ethclient_test.go

```diff
@@ -221,7 +221,11 @@ type mockHistoricalBackend struct{}
 func (m *mockHistoricalBackend) Call(ctx context.Context, args ethapi.TransactionArgs, blockNrOrHash rpc.BlockNumberOrHash, overrides *ethapi.StateOverride) (hexutil.Bytes, error) {
 	num, ok := blockNrOrHash.Number()
 	if ok && num == 100 {
-		return hexutil.Bytes("test"), nil
+		return hexutil.Bytes("testNum"), nil
```

Comment:
Adding tests is great, but if we're modifying upstream tests we increase the diff and merge complexity. Maybe we build an e2e test-suite in the monorepo around the daisychain instead?

Reply:
A very good point. Will address.